setup.py

```diff
@@ -0,0 +1,36 @@
+"""
+A setuptools based setup module.
+
+See:
+https://packaging.python.org/guides/distributing-packages-using-setuptools/
+"""
+
+import setuptools
+
+DATA_FILES = [
+    'particle_list.xml',
+    'particle_list.yaml',
+]
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="tensorwaves",
+    version='0.0-alpha2',
```

Comment:
0.0-alpha. But the whole semantic versioning https://semver.org/ does not really make much sense for us, since we wont bugfix older version and keep older versions alive as well, right?

Reply:
Atm not, but this style is required for and interpreted by PyPI. The alpha2 (instead of alpha1) is because I already uploaded a alpha1.

Anyway, once interfaces stabilise it becomes easy to update the package now:

1. Change the `version` argument of `setuptools.setup` that you see here.
2. Create a PR
3. Squash and approve
4. Create a new tag *from the latest, squashed commit* in the master branch of the form digit.digit(whatever else)
5. Push it to the [ComPWA/tensorwaves](https://github.com/ComPWA/tensorwaves/) repo

Travis will take care of the uploading to PyPI

Reply:
>Atm not, but this style is required for and interpreted by PyPI. 

I see. But why do packages exist right now. Its completely useless right now. Can you delete that version again

Reply:
Idk, just faithfully followed the issue https://github.com/ComPWA/tensorwaves/issues/19

But yes, versions can be deleted afterwards on PyPI

Reply:
Oh and I also needed alpha1 to be able to create a PyPI token :grimacing:

Reply:
I understand you need to create a version for trying it out. But I would just change it back to alpha1 and delete the version on pypi again, since it brings absolutely no benefits (or am I wrong here).
Also currently pypi shows the old info (wrong licence or at least inconsistent).

Reply:
Yes, even after removing PyPI complained the version already existed
https://travis-ci.com/github/ComPWA/pycompwa/jobs/321642688#L3186

So this is why I use 0.0, it's totally for testing and the version will be removed once travis works properly